great_expectations/datasource/fluent/interfaces.py

```diff
@@ -171,8 +172,10 @@ class DataAsset(FluentBaseModel, Generic[_DatasourceT]):
 
     order_by: List[Sorter] = Field(default_factory=list)
     batch_metadata: BatchMetadata = pydantic.Field(default_factory=dict)
+    batch_configs: List[Any] = Field(default_factory=list)
```

Comment:
Is there a more specific type than `Any` that can be used here?

Reply:
Oh thanks! That was a workaround while I dealt with some circular dependencies. Fixed!